site/documentation/content/user-guide/mqtt-adapter.md

```diff
@@ -127,6 +127,12 @@ This requires that
 
 The protocol adapter checks the configured [message limit] ({{< relref "/concepts/resource-limits.md" >}}) before accepting any event messages. If the message limit is exceeded or the incoming event message cannot be processed, the connection to the client is closed.
 
+The devices can optionally indicate a *time-to-live* duration for event messages by setting the *hono-ttl* property explicitly in the `property-bag`. The `property-bag` is an optional collection of properties intended for the receiver of the message. A property bag is only allowed at the very end of a topic. It always starts with a `?` character, followed by pairs of URL encoded property names and values that are separated by `&`. The following example shows a property bag that contains two properties *seqNo* and *importance*:
+
+    /topic/name/?seqNo=10034&importance="high"
```

Comment:
do we actually support quotes? If so, what meaning do they have?
My understanding is that every value in the property bag is considered a string, isn't it?

Reply:
Every value is considered as a string and the above quotes doesn't have a special purpose. I will remove it.